### PR TITLE
Add flex properties

### PR DESCRIPTION
**Pull request by jakezatecky**
In the concentric model, flex properties are part of the box location, and therefore belong near the top. With the current configuration, if you have any flex properties, Stylelint will want you to throw them to the bottom.

I adapted this from the [scss-lint concentric configuration](https://github.com/brigade/scss-lint/blob/master/data/property-sort-orders/concentric.txt).

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -13,6 +13,18 @@ config.rules['declaration-block-properties-order'] = [[
...
   'left',
+  'flex',
+  'flex-basis',
+  'flex-direction',
+  'flex-flow',
+  'flex-grow',
+  'flex-shrink',
+  'flex-wrap',
+  'align-content',
+  'align-items'
```
Comment (danilobuerger) [inline index.js]: Missing comma

**Review by danilobuerger — CHANGES_REQUESTED** [review]

**Comment by danilobuerger** [comment]
Hi @jakezatecky,

Thank you for the PR!

Please amend it with the requested change and i will merge it.

**Comment by jakezatecky** [comment]
Haha, I used the online editor and did not see the missing comma. I added the missing comma and squashed this into one commit.

**Comment by danilobuerger** [comment]
Thanks again. Released in 1.0.7.

**Pull request merged**